styles/_queries.scss

```diff
@@ -0,0 +1,102 @@
+@media screen and (max-width: 600px) {
+    nav {
+        display: flex;
+        width: 100%;
+    }
+    .nav-words {
+        display: flex;
+        color: white;
+        padding: 0 1rem;
+        font-size: 1rem;
+        width: 100%;
+        text-align: center;
+    }
+    h1 {
+        text-align: center;
+        padding: 0rem;
+        margin: 0;
+        font-size: 1rem;
+        width: 100%;
+    }
+    .bigger-hero {
+      padding: 0;
+    }
+    .hero {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        flex-wrap: wrap;
+    }
+    .project-row {
+        display: flex;
+        width: 100%;
```

Comment:
remove this `width: 100%` since you have negative margins, 100% no longer accounts for enough space and is why things are slightly off center in some areas.